integration_tests/README.rst

```diff
@@ -3,13 +3,16 @@ How to run the tests in own environment
 
 Navigate to your Polar2Grid directory, then run::
 
+    conda env update -n jenkins_p2g_docs -f build_environment.yml
+    conda env update -n jenkins_p2g_docs -f jenkins_environment.yml
```

Comment:
Just curious, is it possible to list multiple YAML files to a single `conda env update` call?